labelme/app.py

```
@@ -905,6 +922,36 @@ def __init__(
+            self.imageDataCache[filename] = imageData
+
+    def get_image_data(self, filename):
+        if len(self.imageDataCache) < self.imgDataCacheSize:
+            current_index = self.imageList.index(filename)
+            half_num = self.imgDataCacheSize // 2
+            load_indexes = []
+            front = max(current_index - half_num, 0)
+            load_indexes.extend(list(range(front, current_index)))
+            end = min(current_index + half_num, len(self.imageList) - 1)
+            load_indexes.extend(list(range(current_index + 1, end + 1)))
+            futures = {}
```

Ruff (0.8.2)

Local variable `futures` is assigned to but never used

Remove assignment to unused variable `futures`

(F841)